### ISSUE TITLE
Working with Ava? 

### ISSUE THREAD

**Issue by adack123** [trayio/babel-plugin-webpack-alias#9]
Hi there, 

This plugin looks great and is exactly what I need to use with Ava. Unfortunately I cannot get it to resolve my alias correctly. Here is the code, let me know your thoughts.

In the webpack.config.babel.js:

```javascript
const SRC_DIR = path.join(__dirname, '../src');
....  
resolve: {
        alias: {
            '~': SRC_DIR
        }
    }
```

In the .babelrc 
```javascript
         [ 
          "babel-plugin-webpack-alias",
          { "config": "../build/webpack.config.babel.js" }
         ]
```

In my package.json I'm also requiring 
```javascript 
"require": [
      "babel-register",
      "ignore-styles"
    ]
 ```

Got any ideas? It still is throwing that it can't resolve with that alias. 

```
Can't resolve  ~/components/...
```

**Comment by btkostner** [comment]
I'm not sure if this is the problem you are having, but I was able to get ava working with this configuration:

```json
{
  "presets": [
    "es2015"
  ],

  "plugins": [
    "syntax-async-functions",
    "transform-regenerator",

    ["webpack-alias", {
      "config": "/home/btkostner/Projects/Web/dreamy/build/alias.js"
    }]
  ],

  "comments": false
}
```
Note that I had to use an absolute path because babel was trying to resolve the configuration in the test folder.

```js
/**
 * build/alias.js
 * Holds all the alias information for webpack AND babel :D
 * NOTE: this has to be loadable without babel transcoding!
 *
 * @exports {Object} - a simple webpack configuration with alias
 */

const path = require('path')

const pathSrc = path.resolve(__dirname, '..', 'src')

module.exports = {
  resolve: {
    alias: {
      'assets': path.resolve(pathSrc, 'server', 'assets'),
      'client': path.resolve(pathSrc, 'client'),
      'components': path.resolve(pathSrc, 'client', 'components'),
      'lib': path.resolve(pathSrc, 'lib'),
      'server': path.resolve(pathSrc, 'server')
    }
  }
}
```

```js
/**
 * test/lib/helper/language.js
 * Tests the language helper for accurage usage
 */

import test from 'ava'

import lang from 'lib/helper/language'

test('pad', async (t) => {
  t.is(lang.pad(1, 2), '01')
  t.is(lang.pad(-1, 3), '-01')
  t.is(lang.pad(412, 2), '412')
  t.is(lang.pad(-421, 2), '-421')

  t.throws(() => lang.pad(null, 384))
  t.throws(() => lang.pad(481, null))
})
```

```json
"ava": {
    "require": [
      "babel-register",
      "babel-polyfill"
    ],
    "babel": "inherit"
  }
```
Not sure if the `"babel": "inherit"` is needed as I think it's default.

**Comment by adriantoine** [comment]
Hi!

Sorry for not responding earlier... The plugin doesn't work with Ava at the moment but I'm working on a fix.

I'll make it work like [`babel-plugin-webpack-loaders`](https://github.com/istarkov/babel-plugin-webpack-loaders#dynamic-config-path):
```js
{
  "presets": ["es2015"],
  "env": {
    "AVA": {
      "plugins": [
        [
          "babel-plugin-webpack-alias",
          { "config": "${CONFIG}" }
        ]
      ]
    }
  }
}
```
```console
$ CONFIG=$(pwd)/webpack.config.js NODE_ENV=AVA ava
```

**Comment by adriantoine** [comment]
The fix is included in [v1.6.0](https://github.com/trayio/babel-plugin-webpack-alias/releases/tag/v1.6.0)!
